### PR TITLE
Update New-CsTeamsMessagingPolicy.md

### PR DESCRIPTION
**Pull request by saleens7**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -85,6 +85,38 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AutomaticallyStartCopilot
+
+This policy gives admins the ability to auto-start copilot.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:90]:
```suggestion
This policy gives admins the ability to auto-start Copilot.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -85,6 +85,38 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Copilot
+
+This policy gives admins the ability to choose if copilot will be enabled with a persisted transcript or a non-persisted transcript.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:106]:
```suggestion
This policy gives admins the ability to choose if Copilot will be enabled with a persisted transcript or a non-persisted transcript.
```

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md [hunk 3]
```diff
@@ -60,6 +61,36 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -CreateCustomEmojis
+This setting enables creation of custom emojis/reactions within a tenant for the specified policy users.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:65]:
```suggestion
This setting enables the creation of custom emojis and reactions within an organization for the specified policy users.
```

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md [hunk 4]
```diff
@@ -60,6 +61,36 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DeleteCustomEmojis
+This settings enable/disable the editing/deletion of custom emojis and reactions for the users included in the messaging policy
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:80]:
```suggestion
These settings enable and disable the editing and deletion of custom emojis and reactions for the users included in the messaging policy.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 5]
```diff
@@ -81,6 +81,38 @@ This policy will use all the existing values except one: AllowNetworkConfigurati
 
 ## PARAMETERS
 
+### -AutomaticallyStartCopilot
+
+This policy gives admins the ability to auto-start copilot.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:86]:
```suggestion
This policy gives admins the ability to auto-start Copilot.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 6]
```diff
@@ -81,6 +81,38 @@ This policy will use all the existing values except one: AllowNetworkConfigurati
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Copilot
+
+This policy gives admins the ability to choose if copilot will be enabled with a persisted transcript or a non-persisted transcript.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:102]:
```suggestion
This policy gives admins the ability to choose if Copilot will be enabled with a persisted transcript or a non-persisted transcript.
```

File: skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md [hunk 7]
```diff
@@ -66,6 +65,36 @@ All other policy properties will be left as previously assigned.
 
 ## PARAMETERS
 
+### -CreateCustomEmojis
+This setting enables creation of custom emojis/reactions within a tenant for the specified policy users.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:69]:
```suggestion
This setting enables the creation of custom emojis and reactions within an organization for the specified policy users.
```

File: skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md [hunk 8]
```diff
@@ -66,6 +65,36 @@ All other policy properties will be left as previously assigned.
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DeleteCustomEmojis
+This settings enable/disable the editing/deletion of custom emojis and reactions for the users included in the messaging policy
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:84]:
```suggestion
These settings enable and disable the editing and deletion of custom emojis and reactions for the users included in the messaging policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @saleens7, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@saleens7 Gentle reminder.

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please help. Thanks

**Pull request merged**